lombok/src/test/java/com/baeldung/lombok/builder/singular/BuilderWithSingularSupportForCollectionsUnitTest.java

```diff
@@ -0,0 +1,171 @@
+package com.baeldung.lombok.builder.singular;
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class BuilderWithSingularSupportForCollectionsUnitTest {
+
+    @Test
+    public void canAddMultipleElementsAsNewCollection() throws Exception {
+        Person person = Person.builder()
+                .givenName("Aaron")
```

Comment:
Let's adjust the indent on all of these examples. They're at six spaces over from the parent line. The guideline is two, although four is also acceptable with fluent APIs like this.